forms-flow-nav/src/Navbar.css

```diff
@@ -122,14 +122,13 @@ header nav {
   }
   .main-header {
     flex-wrap: nowrap;
-    gap: 20px;
   }
   .admin-container > .header-container {
     margin-left: -1rem;
   }
   .head-rule {
     margin-top: -25px;
-    margin-left: 25px;
-    margin-right: -50px;
+    margin-left: -15px;
```

Comment:
if there is no margin like margin:none then what will happened

Reply:
no changes to that hr and not in proper width for all headings